src/main/java/me/xemor/configurationdata/entity/EntityData.java

```diff
@@ -136,6 +136,15 @@ public static EntityData create(ConfigurationSection configurationSection) {
         return create(configurationSection, EntityType.ZOMBIE);
     }
 
+    public static EntityData create(ConfigurationSection parentSection, String path, @NotNull EntityType def) {
+        if (parentSection.isString(path)) {
```

Comment:
sorry about nitpicking again I feel bad with the reviews

Add a message if EntityType.valueOf fails. 

try {
  ...
} catch(IllegalArgumentException e) {
  blah blah logging blah blah you are a noob and entered an invalid entity at parentSection.getCurrentPath() + ".entity"
}

Reply:
You're totally fine, it's important to make sure it fits what you want for your plugins as it is your project :)